.github/USING_FEEDBACK_DATA.md

```diff
@@ -0,0 +1,198 @@
+- Antagonist win/loss rates if a new gamemode or antagonist is being added
+- Department performance (IE: Slime cores produced in science)
+- Basically anything which has actual meaning
+
+Examples of bad things to record:
+
+- Amount of times a wrench is used (Why)
+- Hours spent on the server (We have other means of that)
+- Basically, just think about it and ask yourself "Is this actually useful to base game design around"
+
+## Feedback Data Recording
+
+Feedback data can be reocrded in 5 formats. `amount`, `associative`, `nested tally`, `tally` and `text`.
+
+### Amount
+
+`amount` is the simplest form of feedback data recording. They are simply a numerical number which increase with each feedback increment. For example:
+
+These DM calls:
+
+```dm
+SSblackbox.record_feedback("amount", "example", 8)
+SSblackbox.record_feedback("amount", "example", 2)
+// Note that you can use negative increments to decrease the value
+```
+
+Will produce the following JSON:
+
+```json
+{
+    "data":10
+}
+```
+
+Notice the lack of any specific identification other than it being the value of the `data` key. Amounts are designed to be simple with minimal complexity, and are useful for logging statistics such as how many times one specific, distinct action is done (EG: How many MMIs have been filled). If you want to log multiple similar things (EG: How many mechas have been created), use a `tally` with a sub-key for each different mecha, instead of an amount with its own key per mecha.
+
+### Associative
+
+`associative` is used to record text that's associated with multiple key-value pairs. (EG: coordinates). Further calls to the same key will append a new list to existing data. For example:
+
+```dm
+SSblackbox.record_feedback("associative", "example", 1, list("text" = "example", "path" = /obj/item, "number" = 4))
+SSblackbox.record_feedback("associative", "example", 1, list("number" = 7, "text" = "example", "other text" = "sample"))
+```
+
+Will produce the following JSON:
+
+```json
+{
+    "data":{
+        "1":{
+            "text":"example",
+            "path":"/obj/item",
+            "number":"4"
+        },
+        "2":{
+            "number":"7",
+            "text":"example",
+            "other text":"sample"
+        }
+    }
+}
+```
+
+Notice how everything is casted to strings, and each new entry added has its index increased ("1", "2", etc).
+
+### Nested Tally
+
+`nested tally` is used to track the number of occurances of structured semi-relational values (EG: the results of arcade machines). Similar to running total, but related values are nested in a multi-dimensional array. The final element in the data list is used as the tracking key, all prior elements are used for nesting. Further calls to the same key will add or subtract from the saved value of the data key if it already exists in the same multi-dimensional position,  append the key and it's value if it doesn't exist already. This one is quite complicated, but an example is below:
```

Comment:
I find it nicer to think of as just "tally, but the key is a list". Consider:

`nested tally` is used to track the number of occurances of structured semi-relational values (e.g.: the results of arcade machines). You can think of it as a running total, with the key being a list of strings (rather than a single string), with elements incrementally identifying the entity in question.
Technically, the values are nested in a multi-dimensional array. The final element in the data list is used as the tracking key, all prior elements are used for nesting. Further calls to the same key will add or subtract from the saved value of the data key if it already exists in the same multi-dimensional position,  append the key and it's value if it doesn't exist already. This one is quite complicated, but an example is below: